tests/test_py3_syntax.py

```diff
@@ -155,6 +155,10 @@ def test_FormattedValue():
     check("f'{1/3:.1f}'")
     check(r"f'{a}\''")
     check("f'{1/3:{5}.1}'")
+    check("f'{a, b, c}'")
+    check("f'{a: b}'")
+    check("f'{a: b for a, b in ((1,2))}'")
```

Comment:
This isn't actually testing a dictcomp, it's interpolating the variable `a` with a format spec of ` b for a, b in ((1,2))` (which greatly confused me). Maybe we should add a test that actually uses a dictcomp?